src/ansys/mapdl/core/plotting/theme.py

```diff
@@ -25,7 +25,7 @@
 import numpy as np
 
 try:
-    from matplotlib.colors import ListedColormap
+    from matplotlib.colors import LinearSegmentedColormap, ListedColormap
```

Comment:
Why is it in a ``try`` section?

Reply:
Because it depends on having Matplotlib installed, and we want to be able to run certain parts of the library without it. So we do evaluate those imports in a try.